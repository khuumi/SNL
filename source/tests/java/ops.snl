start ops:
	do show to 1 + 2
	do show to 1 - 2
	do show to 1 * 2
	do show to 10 / 2
	do show to 1910
	do show to 1 = 2
	do show to 2 = 2
	do show to 1910
	do show to 1 != 2
	do show to 2 != 2
	do show to 1910
	do show to 1 < 2
	do show to 3 < 2
	do show to 1910
	do show to 1 <= 2
	do show to 1 <= 1
	do show to 2 <= -1
	do show to 1910
	do show to 1 > 2
	do show to 1 < 2
	do show to 1910
	do show to 1 >= 2
	do show to 2 >= 2
	do show to 2 >= 1
done