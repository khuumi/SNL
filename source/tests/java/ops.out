3
-1
2
5
1910
false
true
1910
true
false
1910
true
false
1910
true
true
false
1910
false
true
1910
false
true
true
